python/triton_kernels/bench/bench_mlp.py

```diff
@@ -111,7 +111,7 @@ def roofline_mlp(batch_sizes, dim1, dim2, n_expts_tot, n_expts_act, x_dtype, w_d
 
 
 if __name__ == "__main__":
-    has_native_mx4 = torch.cuda.get_device_capability(0)[0] >= 10 or get_cdna_version() == 4
+    has_native_mx4 = torch.cuda.get_device_capability(0)[0] == 10 or get_cdna_version() == 4
```

Comment:
this is likely to not be correct for the future targets. I'm think we need an explicit helper otherwise we are just pushing the problem to the next target upgrade